src/components/elementView/detailedView/propertiesView.tsx

```diff
@@ -75,20 +80,7 @@ class PropertiesView extends React.PureComponent<
           dat = <p>{network_ports.toString()}</p>;
         }
       } else if (item === "model") {
-        const isDecommissioned = data["decommissioning_user"];
```

Comment:
What's the reason for getting rid of decommissioning stuff here?

Reply:
i removed the onClick logic from the model property because I added a button in assetView.tsx that links to model view page, so i dont need this logic anymore

Reply:
(the decommissioned logic is preserved for the button)